src/exporter.js

```diff
@@ -133,9 +136,22 @@ export default class {
   _tagsToStr(tags=[]){
     return tags.map(tag => tag.replace(/ /g, '_')).join(' ');
   }
-}
 
-export const rand = () => Math.random() * 100000000 | 0;
+  _getId(table, col, ts) {
+    const query = `SELECT ${col} from ${table} WHERE ${col} >= ${ts}`;
```

Comment:
@ewnd9  I miss it cause I have presupposition that we work only with db generated by exporter. Anyway - looks like use `=` instead of `>=` thus we re-query on each increment iteration.  And in such case no need in `ORDER`

And nope - we can't use params for table/column name. I can't find any spec for this statement. But code throws an error. And google says that I can't. The main purpose of prepared statements - optimizations. And I have no idea how they can improve optimization with dynamic tables/columns